src/commands/trigger-workflows-for-all-modified.yml

```diff
@@ -289,7 +290,10 @@ steps:
             running=$(jq -s 'map(select(.status == "running")) | length' ${workflow_status})
             success=$(jq -s 'map(select(.status == "success")) | length' ${workflow_status})
             echo "total: ${total}, running: ${running}, success: ${success}"
-            if (( $total == 0 || $running > 0 )); then
+            if (( $total == 0 && $running == 0 )); then
```

Comment:
running is a derivative of total, checking total == 0 should be enough

Reply:
I'd also check that next_page_token == null

Reply:
+1

Reply:
All pages read in "pagination loop" (line 278).
I think always next_page_token will be null.